[2025-03-20T15:11:17.266+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: transactions_pipeline.create_fraud_table manual__2025-03-20T15:11:01.641454+00:00 [queued]>
[2025-03-20T15:11:17.280+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: transactions_pipeline.create_fraud_table manual__2025-03-20T15:11:01.641454+00:00 [queued]>
[2025-03-20T15:11:17.281+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-03-20T15:11:17.303+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): create_fraud_table> on 2025-03-20 15:11:01.641454+00:00
[2025-03-20T15:11:17.311+0000] {standard_task_runner.py:57} INFO - Started process 3206 to run task
[2025-03-20T15:11:17.314+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'transactions_pipeline', 'create_fraud_table', 'manual__2025-03-20T15:11:01.641454+00:00', '--job-id', '361', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_pipeline.py', '--cfg-path', '/tmp/tmp2l5aeti9']
[2025-03-20T15:11:17.319+0000] {standard_task_runner.py:85} INFO - Job 361: Subtask create_fraud_table
[2025-03-20T15:11:17.388+0000] {task_command.py:415} INFO - Running <TaskInstance: transactions_pipeline.create_fraud_table manual__2025-03-20T15:11:01.641454+00:00 [running]> on host 3938a1dd8bd9
[2025-03-20T15:11:17.516+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='rabiizahnoune7@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='transactions_pipeline' AIRFLOW_CTX_TASK_ID='create_fraud_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-20T15:11:01.641454+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-20T15:11:01.641454+00:00'
[2025-03-20T15:11:17.520+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-20T15:11:17.522+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n        docker exec hive beeline -u "jdbc:hive2://localhost:10000" -e "\n        CREATE EXTERNAL TABLE IF NOT EXISTS fraud_detections (\n            transaction_id STRING,\n            date_time STRING,\n            amount DOUBLE,\n            customer_id STRING,\n            location STRING,\n            fraud_reason STRING\n        )\n        STORED AS PARQUET\n        LOCATION \'/data/fraud_detections\';"\n        ']
[2025-03-20T15:11:17.538+0000] {subprocess.py:86} INFO - Output:
[2025-03-20T15:11:18.816+0000] {subprocess.py:93} INFO - SLF4J: Class path contains multiple SLF4J bindings.
[2025-03-20T15:11:18.817+0000] {subprocess.py:93} INFO - SLF4J: Found binding in [jar:file:/opt/hive/lib/log4j-slf4j-impl-2.17.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
[2025-03-20T15:11:18.818+0000] {subprocess.py:93} INFO - SLF4J: Found binding in [jar:file:/opt/hadoop/share/hadoop/common/lib/slf4j-log4j12-1.7.25.jar!/org/slf4j/impl/StaticLoggerBinder.class]
[2025-03-20T15:11:18.819+0000] {subprocess.py:93} INFO - SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
[2025-03-20T15:11:18.979+0000] {subprocess.py:93} INFO - SLF4J: Actual binding is of type [org.apache.logging.slf4j.Log4jLoggerFactory]
[2025-03-20T15:11:20.000+0000] {subprocess.py:93} INFO - Connecting to jdbc:hive2://localhost:10000
[2025-03-20T15:11:20.241+0000] {subprocess.py:93} INFO - Connected to: Apache Hive (version 3.1.3)
[2025-03-20T15:11:20.242+0000] {subprocess.py:93} INFO - Driver: Hive JDBC (version 3.1.3)
[2025-03-20T15:11:20.242+0000] {subprocess.py:93} INFO - Transaction isolation: TRANSACTION_REPEATABLE_READ
[2025-03-20T15:11:20.358+0000] {subprocess.py:93} INFO - INFO  : Compiling command(queryId=root_20250320151120_2d359738-cfa0-4ddd-9562-9a9f5a983f50): CREATE EXTERNAL TABLE IF NOT EXISTS fraud_detections (
[2025-03-20T15:11:20.360+0000] {subprocess.py:93} INFO -             transaction_id STRING,
[2025-03-20T15:11:20.361+0000] {subprocess.py:93} INFO -             date_time STRING,
[2025-03-20T15:11:20.361+0000] {subprocess.py:93} INFO -             amount DOUBLE,
[2025-03-20T15:11:20.362+0000] {subprocess.py:93} INFO -             customer_id STRING,
[2025-03-20T15:11:20.363+0000] {subprocess.py:93} INFO -             location STRING,
[2025-03-20T15:11:20.363+0000] {subprocess.py:93} INFO -             fraud_reason STRING
[2025-03-20T15:11:20.364+0000] {subprocess.py:93} INFO -         )
[2025-03-20T15:11:20.364+0000] {subprocess.py:93} INFO -         STORED AS PARQUET
[2025-03-20T15:11:20.365+0000] {subprocess.py:93} INFO -         LOCATION '/data/fraud_detections'
[2025-03-20T15:11:20.366+0000] {subprocess.py:93} INFO - INFO  : Concurrency mode is disabled, not creating a lock manager
[2025-03-20T15:11:20.366+0000] {subprocess.py:93} INFO - INFO  : Semantic Analysis Completed (retrial = false)
[2025-03-20T15:11:20.367+0000] {subprocess.py:93} INFO - INFO  : Returning Hive schema: Schema(fieldSchemas:null, properties:null)
[2025-03-20T15:11:20.367+0000] {subprocess.py:93} INFO - INFO  : Completed compiling command(queryId=root_20250320151120_2d359738-cfa0-4ddd-9562-9a9f5a983f50); Time taken: 0.005 seconds
[2025-03-20T15:11:20.368+0000] {subprocess.py:93} INFO - INFO  : Concurrency mode is disabled, not creating a lock manager
[2025-03-20T15:11:20.369+0000] {subprocess.py:93} INFO - INFO  : Executing command(queryId=root_20250320151120_2d359738-cfa0-4ddd-9562-9a9f5a983f50): CREATE EXTERNAL TABLE IF NOT EXISTS fraud_detections (
[2025-03-20T15:11:20.369+0000] {subprocess.py:93} INFO -             transaction_id STRING,
[2025-03-20T15:11:20.370+0000] {subprocess.py:93} INFO -             date_time STRING,
[2025-03-20T15:11:20.370+0000] {subprocess.py:93} INFO -             amount DOUBLE,
[2025-03-20T15:11:20.371+0000] {subprocess.py:93} INFO -             customer_id STRING,
[2025-03-20T15:11:20.373+0000] {subprocess.py:93} INFO -             location STRING,
[2025-03-20T15:11:20.375+0000] {subprocess.py:93} INFO -             fraud_reason STRING
[2025-03-20T15:11:20.376+0000] {subprocess.py:93} INFO -         )
[2025-03-20T15:11:20.377+0000] {subprocess.py:93} INFO -         STORED AS PARQUET
[2025-03-20T15:11:20.377+0000] {subprocess.py:93} INFO -         LOCATION '/data/fraud_detections'
[2025-03-20T15:11:20.378+0000] {subprocess.py:93} INFO - INFO  : Starting task [Stage-0:DDL] in serial mode
[2025-03-20T15:11:20.378+0000] {subprocess.py:93} INFO - INFO  : Completed executing command(queryId=root_20250320151120_2d359738-cfa0-4ddd-9562-9a9f5a983f50); Time taken: 0.03 seconds
[2025-03-20T15:11:20.380+0000] {subprocess.py:93} INFO - INFO  : OK
[2025-03-20T15:11:20.381+0000] {subprocess.py:93} INFO - INFO  : Concurrency mode is disabled, not creating a lock manager
[2025-03-20T15:11:20.382+0000] {subprocess.py:93} INFO - No rows affected (0.108 seconds)
[2025-03-20T15:11:20.382+0000] {subprocess.py:93} INFO - Beeline version 3.1.3 by Apache Hive
[2025-03-20T15:11:20.388+0000] {subprocess.py:93} INFO - Closing: 0: jdbc:hive2://localhost:10000
[2025-03-20T15:11:20.444+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-20T15:11:20.479+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=transactions_pipeline, task_id=create_fraud_table, execution_date=20250320T151101, start_date=20250320T151117, end_date=20250320T151120
[2025-03-20T15:11:20.507+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-20T15:11:20.537+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
